apps/condo/domains/user/schema/ExternalTokenAccessRight.js

```diff
@@ -13,7 +13,8 @@ const { IDENTITY_TYPES, SBBOL_IDENTITY_TYPE } = require('@condo/domains/user/con
 
 
 const ExternalTokenAccessRight = new GQLListSchema('ExternalTokenAccessRight', {
-    schemaDoc: 'The scheme regulates the access of the service user to the access tokens stored in the redis',
+    schemaDoc: 'The scheme allows service user access to the access tokens of all users of the selected integration stored in redis.\n' +
```

Comment:
```suggestion
    schemaDoc: 'The scheme allows the service user to get read access to all user access tokens.\n' +
```

you do not need to describe some technical details related to storage in the docs.